lib/snapshot.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+const path = require('path');
+const fs = require('fs');
+const isEqual = require('lodash.isequal');
+const mkdirp = require('mkdirp');
+const globals = require('./globals');
+
+class Snapshot {
+	constructor(testPath, options) {
+		if (!testPath) {
+			throw new TypeError('Test file path is required');
+		}
+
+		this.dirPath = path.join(path.dirname(testPath), '__snapshots__');
```

Comment:
Jest convention aside, is there a reason we should stick to this format? Our [naming scheme is `helpers` and `fixtures`](https://github.com/avajs/ava/blob/314ef001ab1c085e2057738e2d2588bde3e792cc/lib/ava-files.js#L76:L80), not `__fixtures__`.